Gem/Code/CMakeLists.txt

```diff
@@ -60,11 +62,95 @@ ly_add_target(
             Gem::Atom_AtomBridge.Static
 )
 
+ly_add_target(
```

Comment:
It seems a bit weird that these modules aren't defined in the gamelift gem itself with a config file or settings registry to drive most game specific behaviour? Then the integration into mpsample would be vastly lighter, the gamelift gem would provide significantly more value out of the box, and also for customers wanting to bring the gamelift gem into their own project they wouldn't be copy-pasting so much code out of mpsample.